packages/web-app-admin-settings/src/components/Users/ContextActions.vue

```diff
@@ -66,6 +61,25 @@ export default defineComponent({
       { deep: true, immediate: true }
     )
 
+    const usersWithoutDrive = computed(() => {
+      return props.items.filter(({ drive }) => !isPersonalSpaceResource(drive as SpaceResource))
```

Comment:
Could you please sort them by displayName?

Reply:
wow missed that